examples/src/main/java/io/dapr/examples/configuration/grpc/ConfigurationClient.java

```diff
@@ -145,11 +201,11 @@ private static void print(ConfigurationItem item, String key) {
     System.out.println(item.getValue() + " : key ->" + key);
   }
 
-  private static void executeDockerCommand(int postfix) {
+  private static void executeDockerCommand(String key, int postfix) {

```

Comment:
Please, move this `docker exec` logic to README.md. This is not a good example to follow for a real app.

Reply:
removed